cmd/contour/serve.go

```diff
@@ -503,7 +527,10 @@ func doServe(log logrus.FieldLogger, ctx *serveContext) error {
 	g.Add(lbsw.Start)
```

Comment:
similar to the main contour event handler, I *think* that we should probably be waiting for the informer caches to sync in here before starting to process events to avoid any race conditions with the caches populating, but I'd have to experiment a bit to confirm if needed/not problematic.

Reply:
???

Reply:
Yeh, that's a different issue though.

Reply:
filed #3020